src/app/components/calendar/calendar.ts

```diff
@@ -1026,6 +1026,7 @@ export class Calendar implements OnInit, OnDestroy, ControlValueAccessor {
         this.createResponsiveStyle();
         this.currentMonth = date.getMonth();
         this.currentYear = date.getFullYear();
+        this.yearOptions = [this.currentYear, this.currentYear + 1, this.currentYear + 2];
```

Comment:
Hi, What is the reason why you think the `yearOptions` value should start this way?

Reply:
> Hi, What is the reason why you think the `yearOptions` value should start this way? @SoyDiego

Hi @gucal, thanks for the double check.
I don't remember exactly why I did that because was 1 month ago, but I was checking now and I

Reply:
Thank you for making the changes.

Thanks for contribution.